uniflow/op/model/layout_utils.py

```diff
@@ -0,0 +1,348 @@
+"""Layout Module."""
+from typing import List
+
+import cv2
+import numpy as np
+import onnxruntime
+from huggingface_hub import hf_hub_download
+
+
+def preprocess(img, input_size, swap=(2, 0, 1)):
+    """
+    Preprocesses the input image by resizing and padding it.
+
+    Args:
+        img (numpy.ndarray): The input image.
+        input_size (tuple): The desired size of the input image.
+        swap (tuple, optional): The order of dimensions to swap. Defaults to (2, 0, 1).
+
+    Returns:
+        numpy.ndarray: The preprocessed image.
+        float: The scaling factor applied to the image.
+    """
+    if len(img.shape) == 3:
+        padded_img = np.ones((input_size[0], input_size[1], 3), dtype=np.uint8) * 114
```

Comment:
qq: curious about why `114` is the default padding value?

Reply:
yolox default padding value is 114
https://github.com/Megvii-BaseDetection/YOLOX/blob/main/demo/OpenVINO/cpp/yolox_openvino.cpp#L36